containerization-installs/docker-install.sh

```diff
@@ -65,4 +65,4 @@ then
 else
   echo "Installing docker"
   sudo apt install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
```

Comment:
I would break each package install into an apt-cache show check and then apt install.

Reply:
Yes that sounds like a plan.